src/protocol.py

```diff
@@ -383,13 +430,16 @@ def checkAndShareObjectWithPeers(data):
     if not isProofOfWorkSufficient(data):
         logger.info('Proof of work is insufficient.')
         return 0
-    
+
     endOfLifeTime, = unpack('>Q', data[8:16])
-    if endOfLifeTime - int(time.time()) > 28 * 24 * 60 * 60 + 10800: # The TTL may not be larger than 28 days + 3 hours of wiggle room
+    # The TTL may not be larger than 28 days + 3 hours of wiggle room
+    if endOfLifeTime - int(time.time()) > 28 * 24 * 60 * 60 + 10800:
         logger.info('This object\'s End of Life time is too far in the future. Ignoring it. Time is %s', endOfLifeTime)
         return 0
-    if endOfLifeTime - int(time.time()) < - 3600: # The EOL time was more than an hour ago. That's too much.
-        logger.info('This object\'s End of Life time was more than an hour ago. Ignoring the object. Time is %s', endOfLifeTime)
+    if endOfLifeTime - int(time.time()) < - 3600:  # The EOL time was more than an hour ago. That's too much.
```

Comment:
This was already there, in 2.7.13 it seems to do the right thing. Again, raise an issue and I'll check the history of this, perhaps older versions we still support handle it differently so I'm reluctant to change it as part of a style / lint commit.

```
>>> - 4 < - 3
True
>>> - 2 < - 3
False
```

Reply:
My remark was exactly about style: I think the space is redundant here. No doubt that works correctly in any python interpreter.

Reply:
No doubt? The linters didn't pick it up. Nor autopep8. So `-3600` can't be the canonical version. I'm going assume they know something I don't until I investigate. They have a lot of bugs though, so this could be one of them. Please, raise an issue, assign it to me and I promise I will investigate.

The style I was aiming for here (at Peter's direction) was one that passed codacy but evolved (via github discussion) into one that passes multiple linters. 

I don't want to bike-shed about things like hanging brackets in lists/dicts and functions (I like de-dented closes for data and long functions and trailing ones for short functions) or single line #-comments above a line vs a triple-quoted string below (sphinx likes the latter) or the number of statements between single blank lines (I like about 6).

I'm all in favour of us voting on which further, voluntary conventions to stick to. And I will stick to the ones that I don't even like for the sake of the consistency of the project. A consistent project is better than an inconsistent project. But then again, a happy developer is better than an unhappy developer ;)